forum_leaderboard/static/js/src/forum_leaderboard_studio.js

```diff
@@ -0,0 +1,21 @@
+/* Javascript for ForumLeaderboardXBlock. */
+function ForumLeaderboardStudioXBlock(runtime, element) {
+    $('.save-button', element).bind('click', function() {
+        var handlerUrl = runtime.handlerUrl(element, 'studio_submit');
+        var data = {
+            discussion_id: $(element).find('input[name=discussion_id]').val(),
+            count: $(element).find('input[name=count]').val()
+        };
+        $.post(handlerUrl, JSON.stringify(data)).success(function(response) {
+            if (response['success']) {
+                window.location.reload(false);
+                return
+            }
+            alert(response['errors'].join('\n'));
```

Comment:
@Kelketek You can clean this up: raise a `xblock.exceptions.JsonHandlerError(status_code, message)` in your handler, and then you have nice HTTP compliant responses instead of having to look in the JSON.

[source](http://xblock.readthedocs.org/en/latest/api/exceptions.html)